assets/brunch-config.js

```diff
@@ -0,0 +1,70 @@
+const neat = require('bourbon-neat').includePaths;
+const bourbon = require('bourbon').includePaths;
+
+exports.config = {
+  // See http://brunch.io/#documentation for docs.
+  files: {
+    javascripts: {
+      joinTo: "js/app.js"
```

Comment:
Strings must use singlequote.

[Read more about it here.](http://eslint.org/docs/rules/quotes)